### PR TITLE
refactor(wasm-gen): Remove prints from `gear-wasm-gen`

### PR DESCRIPTION
**Pull request by SabaunT**

### CODE REVIEW

**Review by StackOverflowExcept1on — APPROVED** [review]

File: utils/wasm-gen/src/config/syscalls/param.rs [hunk 1]
```diff
@@ -54,7 +54,7 @@ impl Default for SysCallsParamsConfig {
                 (ParamType::Ptr(None), (0..=513 * 0x10000 - 1).into()),
                 (ParamType::Gas, (0..=250_000_000_000).into()),
                 (ParamType::MessagePosition, (0..=10).into()),
-                (ParamType::Duration, (0..=100).into()),
+                (ParamType::Duration, (0..=10).into()),
```
Comment (breathx) [inline utils/wasm-gen/src/config/syscalls/param.rs:57]:
```suggestion
                (ParamType::Duration, (1..=8).into()),
```

**Review by breathx — APPROVED** [review]

**Comment by breathx** [comment]
merge master to fix CI

File: utils/wasm-gen/src/config/syscalls/param.rs (hunk 1 above)
Comment (SabaunT) [reply utils/wasm-gen/src/config/syscalls/param.rs]: done

**Comment by shamilsan** [comment]
Great job: add `println`, then remove them 😄

**Pull request merged**